src/SurveyService.Data.Provider.MsSql.Ef/Migrations/20220826180800_InitialQuestionsTable.cs

```diff
@@ -0,0 +1,40 @@
+﻿using LT.DigitalOffice.SurveyService.Models.Db;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+
+namespace LT.DigitalOffice.SurveyService.Data.Provider.MsSql.Ef.Migrations;
+
+[DbContext(typeof(SurveyServiceDbContext))]
+[Migration("20220826180800_InitialQuestionsTable")]
```

Comment:
While service is not in release, better have one migration for initial tables - it is easier to fix and read; other tables can just would be added to this migration.
So, please, change name to "InitialCreate"

Reply:
Done